### DISCUSSION TITLE
Animation on Linked Character?

### DISCUSSION THREAD

**Discussion by degrootmedia** [soupday/cc3_blender_tools#122] [Q&A]
Hi! When linking a character into a new scene, the option for retargeting animation doesn't appear - it says "No Current Character" even when I have a library override on the rig. Is there a way around this? I'd love to be able to link character rigs into different projects while still being able to use the addon's animation tools.
Amazing addon by the way!

**Comment by boxofrabbits** [comment]
Hello, I'd like to back this one too please. Being able to swap out characters with Linked Library Overrides would be brilliant. Is there a way to manually apply the FBX decrypt key or something?

**Comment by micahganske** [comment]
Would also love to know if this is possible! This would really smooth out the pipeline.
